python/setup.py

```diff
@@ -442,13 +465,15 @@ def has_ext_modules(self):
             return True
 
 
+# Ensure no remaining lib files.
+shutil.rmtree(os.path.join(ROOT_DIR, "build"))
```

Comment:
when do we generate this build dir?

Reply:
"setup.py" will generate "build" dir but don't delete it. And if you rerun this script, you will include a lot of old files from "build" dir.

Reply:
ok got it